rest_framework_jwt/utils.py

```diff
@@ -130,6 +130,9 @@ def jwt_response_payload_handler(token, user=None, request=None):
         }
 
     """
-    return {
+    data = {
         'token': token
     }
+    if api_settings.JWT_PERMANENT_TOKEN_AUTH:
+        data['permanent_token'] = permanent_token
```

Comment:
why you return always permanent token here? It should be returned only upon login not always.